### PR TITLE
feat: Use custom ApiHealthReport in Azure Functions HTTP trigger project template

### PR DESCRIPTION
**Pull request by vtermaat**
Use custom ApiHealthReport in Azure Functions HTTP trigger project template
Closes https://github.com/arcus-azure/arcus.templates/issues/600

### CODE REVIEW

File: src/Arcus.Templates.AzureFunctions.Http/HealthFunction.cs [hunk 1]
```diff
@@ -72,12 +73,14 @@ public class HealthFunction : HttpBasedAzureFunction
                 }
 
                 HealthReport healthReport = await _healthCheckService.CheckHealthAsync(cancellation);
+                ApiHealthReport apiHealthReport = ApiHealthReport.FromHealthReport(healthReport);
```
Comment (stijnmoreels) [inline src/Arcus.Templates.AzureFunctions.Http/HealthFunction.cs:76]: Great! This will help us a lot. 👍

**Review by stijnmoreels — COMMENTED** [review]
This is already great, thx!
We should, however, make sure that the new custom model does what it should do and that is remove the exception details from the health report.
You could look at the [existing health report integration tests](https://github.com/arcus-azure/arcus.templates/blob/master/src/Arcus.Templates.Tests.Integration/AzureFunctions/Http/Health/HealthChecksTests.cs) to get some inspiration. Let us know if you experience any problems and thanks again! ❤️

**Comment by stijnmoreels** [comment]
Will take this PR over from @vtermaat .

**Review by stijnmoreels — APPROVED** [review]
Hihi, now I'm approving my own changes 😄

**Comment by fgheysels** [comment]
> Hihi, now I'm approving my own changes 😄

That's cowboying in a sneaky way.

**Review by fgheysels — APPROVED** [review]

**Pull request merged**